ndd_slurm.py

```diff
@@ -184,42 +190,87 @@ def get_slaves(args):
         raise ValueError('one of -D or -d must be specified')
 
 
-def run_slave(args):
-    procs = []
+def run_slave_tar(args, procs):
```

Comment:
Should be something like `start_slave_tar`, because `run` assumes that the function runs entire process inside, but this function only starts it.